python/taichi/ui/scene.py

```diff
@@ -141,16 +165,21 @@ def particles(self,
                 values. If `per_vertex_color` is provided, this is ignored.
             per_vertex_color (Tuple[float]): a taichi 3D vector field, where each
                 element indicate the RGB color of a particle.
-            two_sided (bool): whether or not the triangles should be able to be
```

Comment:
`two_sided` deleted?

Reply:
the note of two_sided is redundant cause there is not two_sided args in Scene.particles method. Maybe I think somebody misinput the note from Scene.mesh which include two_sided arg.